Lib/test/test_concurrent_futures.py

```diff
@@ -178,6 +178,20 @@ def test_thread_names_default(self):
             self.assertRegex(t.name, r'^.*ThreadPoolExecutor.*_[0-4]$')
             t.join()
 
+    def test_default_max_queue_size(self):
+        executor = futures.ThreadPoolExecutor()
+        self.assertEqual(executor._work_queue.maxsize, 0)
+
+    def test_custom_max_queue_size(self):
+        # test custom valid and invalid values
+        for i in range(-10, 10):
+            executor = futures.ThreadPoolExecutor(max_queue_size=i)
```

Comment:
Nothing really, but since the `Queue` constructor doesn't check for validity of arguments I wanted to add a test for invalid arguments.